pcdscalc/common.py

```diff
@@ -1,8 +1,34 @@
 """Module that holds common calculations."""
 import numpy as np
+from ophyd import EpicsSignal
 
 from .constants import WAVELENGTH_TO_ENERGY_LAMBDA, units
 
+try:
+    _energy_pv = EpicsSignal('SIOC:SYS0:ML00:AO627', name='lcls_energy')
```

Comment:
I think this line is problematic.

1. We're establishing an EPICS connection on import. Is it possible to create the energy pv on first use of `get_energy` instead?
2. Does this PV change between lines/between NC and SC?

Reply:
Good points! Yes we can create that pv in the get_energy function instead.
Not sure about the SXR PV, I'd need to check that

Reply:
For what it's worth, I wouldn't mind putting off "making it work for both accelerators" for later, given that you can just put in the raw value you want to use.

Reply:
I've changed it, let me know what you think.
For the energy PV selection I am not sure what would be the best. Look at the hostname and only make the PV when we run from one of the control room machine might just be enough here.